lighthouse-cli/test/smokehouse/byte-efficiency/expectations.js

```diff
@@ -133,12 +133,12 @@ const expectations = [
               },
               {
                 url: 'http://localhost:10200/byte-efficiency/script.js?gzip=1',
-                transferSize: 1158,
+                transferSize: '1100 +/- 100',
```

Comment:
Be a bit more forgiving of transferSize fluctuations. The magnitude is what's important here, not the exact number of bytes.

Reply:
this was done already in a different pr